src/_component/about-me/about-me.scss

```diff
@@ -37,16 +37,16 @@ button.pill i.icon__download {
 }
 
 button[class^="pill"] i[class^="icon__"]{
```

Comment:
I was suggested to have an empty space just after `]` , again, not a big issue at all ✌️

Reply:
thank you 💯